cliquet/__init__.py

```diff
@@ -71,6 +71,7 @@
     'cliquet.sentry_url': None,
     'cliquet.sentry_projects': '',
     'cliquet.statsd_url': None,
+    'cliquet.statsd_prefix': 'cliquet'
```

Comment:
Do you want to add the env here directly or can we use it automatically after the prefix?

Shoudn't we use the project_name directly?